crates/kubelet/src/container/status.rs

```diff
@@ -89,50 +75,10 @@ impl ContainerStatus {
             // handle probes, this should be updated accordingly
             started: Some(true),
             // The rest of the items in status (see docs here:
-            // https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#containerstatus-v1-core)
+            // https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#status-v1-core)
```

Comment:
I think this still needs to be `containerstatus`

Reply:
Good catch. Must’ve been a blanket replace all and I missed that replacement